tests/scripts/github-action-helper.sh

```diff
@@ -221,7 +222,7 @@ EOF
   openssl req -new -key "${DIR}"/"${SERVICE}".key -subj "/CN=${SERVICE}.${NAMESPACE}.svc" -out "${DIR}"/server.csr -config "${DIR}"/csr.conf
 
   export CSR_NAME=${SERVICE}-csr
-
+# shellcheck disable=SC2002,2086
```

Comment:
Why 2086 here? the "${DIR}" is already double quoted.

Please add more of a description like "cat-ing a heredoc into a file is not a useless cat"